### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by t3chguy**
missing documentation

### CODE REVIEW

File: src/github.com/t3chguy/riot-static/register-guest/register-guest.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Michael Telatynski <7t3chguy@gmail.com>
```
Comment (Kegsay) [inline src/github.com/t3chguy/riot-static/register-guest/register-guest.go]: Go style: Files which generate binaries generally live in `cmd/nameofbinary/main.go`. I'd do the same here.

File: src/github.com/t3chguy/riot-static/register-guest/register-guest.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+	if err != nil {
+		return err
+	}
+
+	return ioutil.WriteFile(configPath, configJson, 0600)
+}
+
+func main() {
+	configPath := flag.String("config-file", "./config.json", "The path to the desired config file.")
```
Comment (Kegsay) [inline src/github.com/t3chguy/riot-static/register-guest/register-guest.go]: Please add a `config.sample.json` so people know what the keys are.

**Comment by Kegsay** [comment]
Otherwise LGTM!

File: src/github.com/t3chguy/riot-static/register-guest/register-guest.go (hunk 1 above)
Comment (t3chguy) [inline src/github.com/t3chguy/riot-static/register-guest/register-guest.go]: Done :_

**Pull request merged**